packages/stex-react-renderer/src/lib/SubProblemAnswer.tsx

```diff
@@ -252,18 +252,19 @@ export function SubProblemAnswer({
               borderRadius: '5px',
             }}
           >
-            <MystViewer content={answer || '*Unanswered*'} />
+            <MdViewer content={answer || '*Unanswered*'} />
           </Box>
         ) : (
           <Box display="flex" alignItems="flex-start">
             <Box flexGrow={1}>
-              <MystEditor
-                editorProps={{ border: canSaveAnswer ? '2px solid red' : undefined }}
-                name={`answer-${questionId}-${subProblemId}`}
-                placeholder={'...'}
-                value={answer}
-                onValueChange={onAnswerChanged}
-              />
+              <Box sx={{ border: canSaveAnswer ? '2px solid red' : undefined }}>
```

Comment:
I don't think this is equivalent. This will result in the red box to be also around the 'preview' box. With editor props, the red box will be only around the input box.